tests/nightly/nightly-kickoff.sh

```diff
@@ -260,3 +260,23 @@ EOT
 # Emails an HTML report of the test run results using SendMail.
 sendmail -t < nightly_mail.html
 fi
+
+# Saves test results to reporting server
+testresultsdb="vic-nightly.db"
+rm $testresultsdb
```

Comment:
Is this `rm` command removing the `testresultsdb` you just assigned?  @mhagen-vmware